MapboxCoreNavigation/RouteController.swift

```diff
@@ -720,6 +722,23 @@ extension RouteController: CLLocationManagerDelegate {
             strongSelf.delegate?.routeController?(strongSelf, didRerouteAlong: route)
         }
     }
+    
+    func checkForUpdates() {
+        #if TARGET_IPHONE_SIMULATOR
+            guard let latestVersion = String(describing: Bundle(for: RouteController.self).object(forInfoDictionaryKey: "CFBundleShortVersionString")) else { return }
+            URLSession.shared.dataTask(with: URL(string: "https://www.mapbox.com/mapbox-navigation-ios/navigation/latest_version")!, completionHandler: { (data, response, error) in
```

Comment:
We should put something at this URL before releasing with this PR.

Reply:
Added.

Reply:
https://www.mapbox.com/mapbox-navigation-ios/navigation/latest_version